### PR TITLE
fixed make run-notebook

### PR DESCRIPTION
**Pull request by funlyonions**
I worked on fixing the run-notebook command. There were issues while building the dockerfile supplied. I made modifications to get it working on Ubuntu version: 18.04 an up.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,13 +1,21 @@
 FROM python:3.6
 
-WORKDIR /
+WORKDIR /tensforforce
```
Comment (Daniel-Rafique) [inline Dockerfile]: Should this be tensortrade ?

File: Dockerfile [hunk 2]
```diff
@@ -1,13 +1,21 @@
...
+COPY . ./
+
 
 RUN apt-get update -y && \ 
   apt-get install pandoc -y && \
   apt-get install python-mpi4py -y
 
-RUN pip install --upgrade pip
-RUN pip install -e .[tf,docs,tests,baselines,tensorforce,ccxt,fbm]
-RUN pip install -r /requirements.txt
```
Comment (Daniel-Rafique) [inline Dockerfile]: Shouldn't this be ./requirements.txt

File: Dockerfile [hunk 3]
```diff
@@ -1,13 +1,21 @@
...
-RUN pip install -r /examples/requirements.txt
```
Comment (Daniel-Rafique) [inline Dockerfile]: Same as last comment

File: Dockerfile (hunk 2 above)
Comment (funlyonions) [reply Dockerfile]: It is

File: Dockerfile (hunk 1 above)
Comment (notadamking) [reply Dockerfile]: Why is this here?

File: Dockerfile [hunk 4]
```diff
@@ -1,13 +1,21 @@
...
 
+
+
+RUN pip install --upgrade pip
+RUN pip3 install .[tf,docs,tests,baselines,tensorforce,ccxt,fbm]
+RUN ls -la
```
Comment (notadamking) [inline Dockerfile]: What is the point of this?

File: Dockerfile [hunk 5]
```diff
@@ -1,13 +1,21 @@
...
+RUN pip3 install -r ./requirements.txt
+RUN pip3 install -r ./examples/requirements.txt
+RUN pip3 install mpi4py
```
Comment (notadamking) [inline Dockerfile]: This is already installed by `[tensorforce]` in extra requirements.

**Comment by notadamking** [comment]
I was feeling nice, so I fixed the issues for you and merged. Let me know if this works for you.

**Pull request merged**